pkgs/tools/security/yubikey-agent/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, lib, fetchgit, buildGoModule, libnotify, makeWrapper, pcsclite, pinentry, darwin }:
+
+let pinentryBinaryPath = pinentry.binaryPath or "bin/pinentry";
+
+in buildGoModule rec {
+  pname = "yubikey-agent";
+  version = "0.1.3";
+
+  src = fetchgit {
+    url = "https://filippo.io/yubikey-agent";
+    rev = "v${version}";
+    sha256 = "07gix5wrakn4z846zhvl66lzwx58djrfnn6m8v7vc69l9jr3kihr";
+  };
+
+  buildInputs =
+    lib.optional stdenv.isLinux (lib.getDev pcsclite)
+    ++ lib.optional stdenv.isDarwin (darwin.apple_sdk.frameworks.PCSC);
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  overrideModAttrs = (_: {
+    # patch go-piv to read PCSC headers from the right place
```

Comment:
Please link to https://github.com/go-piv/piv-go/pull/75 here, so we know this will eventually get fixed and can be removed.

Reply:
:+1: I may even just patch it to use go-piv/piv-go#75 directly, using a `go.mod` `replace` directive, because it's a better solution.

Reply:
Still you might want to add the note, so people know when to remove it again.